config/boot.rb

```diff
@@ -33,6 +34,38 @@
 puts "Loading config from #{env_file}"
 require env_file
 
+require 'config'
+Config.setup do |config|
+  # Name of the constant exposing loaded settings
+  config.const_name = 'Settings'
+  # Load environment variables from the `ENV` object and override any settings defined in files.
+  #
+  config.use_env = true
+
+  # Define ENV variable prefix deciding which variables to load into config.
+  #
+  config.env_prefix = 'SETTINGS'
+
+  # What string to use as level separator for settings loaded from ENV variables. Default value of '.' works well
+  # with Heroku, but you might want to change it for example for '__' to easy override settings from command line, where
+  # using dots in variable names might not be allowed (eg. Bash).
+  #
+  config.env_separator = '__'
+end
+
+Config.load_and_set_settings(
+  Config.setting_files(File.expand_path(__dir__), environment)
+)
+
+module Was
+  def self.connect_dor_services_app
+    Dor::Services::Client.configure(url: Settings.dor_services.url,
+                                    token: Settings.dor_services.token)
+  end
+end
+
+Was.connect_dor_services_app
```

Comment:
I'm curious about why you wrapped `Dor::Services::Client.configure(url: Settings.dor_services.url, token: Settings.dor_services.token)` in this method. Do you think it will be used elsewhere or otherwise provides some useful abstraction?

Reply:
Copied from https://github.com/sul-dlss/common-accessioning/blob/master/config/boot.rb#L48-L55  for consistency

Reply:
@jcoyne :ok_man: